bindata/etcd/pod.yaml

```diff
@@ -10,6 +10,35 @@ metadata:
     revision: "REVISION"
 spec:
   initContainers:
+    - name: etcd-ensure-env-vars
+      image: ${IMAGE}
+      imagePullPolicy: IfNotPresent
+      terminationMessagePolicy: FallbackToLogsOnError
+      command:
+        - /bin/sh
+        - -c
+        - |
+          #!/bin/sh
+          set -euo pipefail
+
+          if [ -z "$NODE_NODE_ENVVAR_NAME_ETCD_URL_HOST"]; then
+            echo "env var $NODE_NODE_ENVVAR_NAME_ETCD_URL_HOST is not set"
+            exit 1;
+          fi
+          if [ -z "$NODE_NODE_ENVVAR_NAME_ETCD_NAME"]; then
```

Comment:
Again needs space before the `]`

Reply:
@retroflexer https://github.com/openshift/cluster-etcd-operator/pull/284#discussion_r399206436 gave me enough pointers :) but thanks for the review